### PR TITLE
Move GetPortURL to ingress e2e fw

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

In e2e test framework, GetPortURL() is called in ingress e2e test
framework only. So this moves the function into ingress
e2e test framework to make the dependency simple.

One small change: capitalize error string

**Which issue(s) this PR fixes**:

Ref: #81232

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/framework/ingress/ingress_utils.go [hunk 1]
```diff
@@ -106,6 +106,14 @@ const (
 	// names of GCP resources such as forwarding rules, url maps, target proxies, etc
 	// that it created for the corresponding ingress.
 	StatusPrefix = "ingress.kubernetes.io"
+
+	// poll is how often to Poll pods, nodes and claims.
+	poll = 2 * time.Second
```
Comment (gavinfish) [inline test/e2e/framework/ingress/ingress_utils.go]: Is there a better place for these const 🤔 
They are also used by some other methods in e2e/framework/node, and I just temporarily duplicate them.

**Review by oomichi — COMMENTED** [review]
/approve

File: test/e2e/framework/ingress/ingress_utils.go (hunk 1 above)
Comment (alejandrox1) [reply test/e2e/framework/ingress/ingress_utils.go]: I would recommend to give the const a more specific name for this package.
There isn't a lot more that we can do with it.

**Review by alejandrox1 — COMMENTED** [review]
/priority backlog
/milestone v1.18

also, for context: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87512/pull-kubernetes-e2e-gce/1222191463002017793/ that is the current failure
/retest

File: test/e2e/framework/ingress/ingress_utils.go (hunk 1 above)
Comment (gavinfish) [reply test/e2e/framework/ingress/ingress_utils.go]: These values will not be exported, so I think the names are fine.

**Pull request merged**